module-apps/application-calendar/include/application-calendar/data/CalendarData.hpp

```diff
@@ -46,7 +46,7 @@ class DayMonthData : public gui::SwitchData
         return dateFilter;
     };
 
-    virtual void setData(std::string dayMonthText, const TimePoint &dateNumb)
+    virtual void setData(const std::string &dayMonthText, const TimePoint &dateNumb)
```

Comment:
I think it's worth checking if the string being passed here is not copied n-times before it reaches this function :P

Reply:
It looks that this function is not used anywhere currently ;)